script/company-page.js

```diff
@@ -0,0 +1,87 @@
+"use strict";
+
```

Comment:
remove dash (-) from the file name. This is somehting that can cause unexpected problems.